applications/FluidDynamicsApplication/custom_utilities/fluid_characteristic_numbers_utilities.cpp

```diff
@@ -0,0 +1,376 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "includes/cfd_variables.h"
+#include "utilities/element_size_calculator.h"
+#include "utilities/geometry_utilities.h"
+#include "utilities/parallel_utilities.h"
+
+// Application includes
+#include "fluid_dynamics_application_variables.h"
+#include "fluid_characteristic_numbers_utilities.h"
+
+
+namespace Kratos
+{
+
+    void FluidCharacteristicNumbersUtilities::CalculateLocalCFL(ModelPart& rModelPart)
```

Comment:
ah ok I see why it cannot be const
Maybe then change the name of the fct to `CalculateLocalCFLNumberOnElements`?

Reply:
mmmm... we can, but this would imply changing the API. Considering that we are changing the location of the method, I'd prefer to incrementally introduce these kind of changes.